packages/react-native/scripts/generate.js

```diff
@@ -135,7 +135,7 @@ function generate({ configPath, absolute = false, useJs = false }) {
     updateView(global.view, annotations, normalizedStories, ${options});
   }
 
-  export const view = global.view;
+  export const view: ReturnType<typeof start> = global.view;
```

Comment:
to support the js option what about something like this?

suggestion
  export const view${useJs ? '' : ': ReturnType<typeof start>'} = global.view;
```